public-resources/EvaChecks.csv

```diff
@@ -105,13 +105,18 @@ dq,Warning,Project Overlaps,"3.10 - Project Start Date, 3.11 - Project Exit Date
 pdde,High Priority,Missing Geography Information,2.03.8 - Continuum of Care Information: Geography Type,Please ensure geography information for projects is complete.,"Missing geography information that makes this a High Priority Error are geocode, CoC code, and geography type.",5
 pdde,High Priority,Overlapping CE Participation Records,2.09 CE Participation Status,This project has more than one CE Participation record that covers the same time period. Please be sure you are ending any records that are no longer accurate before creating a new record.,,128
 pdde,High Priority,Overlapping HMIS Participation Records,2.08 HMIS Participation,This project has more than one HMIS Participation record that covers the same time period. Please be sure you are ending any records that are no longer accurate before creating a new record.,,131
+pdde,High Priority,Client-level data in VSP organization,2.01.3 Victim Service Provider,Regardless of HMIS Participation Status, projects under Organizations marked as Victim Service Providers should not have client data in HMIS.,,139
```

Comment:
You need to wrap the "Regardless of HMIS Participation Status, projects..." with quotation marks (from within a simple text editor, like Notepad. Otherwise, the comma there will throw off interpreters reading the file as a comma-delimited file; they'll think it's a delimiter. You can see this if you open the CSV in Excel and view that row.

Reply:
Done